backend/src/main/java/com/trainme/app/user_profile/entity/UserProfile.java

```diff
@@ -0,0 +1,33 @@
+package com.trainme.app.user_profile.entity;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+@RequiredArgsConstructor
+@Getter
+@Setter
+@Entity
+public class UserProfile {
+    @Id
+    @GeneratedValue
```

Comment:
As I mentioned before, this should not be a generated value, it should be the same as user ID.